### PR TITLE
Fix merge conflicts from Streaming Support PR on branch datav2

### PR DESCRIPTION
**Pull request by EricWohlstadter**
## What changes were proposed in this pull request?

Fixed merge conflicts on PR that had been based on diverged branch `datav2`.

## How was this patch tested?

`TestHiveStreaming ` example in patch.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -281,7 +329,7 @@ artifact in (Compile, assembly) := {
 }
 addArtifact(artifact in (Compile, assembly), assembly)
 
-resolvers += "Local Maven Repository" at "file://"+Path.userHome.absolutePath+"/.m2/repository"
+resolvers += "Local Maven Repository" at "file:///grid/2/ewohlstadter/.m2/repository"
```
Comment (HyukjinKwon) [inline build.sbt]: Eh, seems a mistake?

File: src/test/scala/com/hortonworks/spark/sql/hive/streaming/TestHiveStreaming.scala [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        .option("keytab", keytab)
+    }
+
+    val query = writer.start()
+    query.awaitTermination()
+
+    query.stop()
+    sparkSession.stop()
+  }
+}
```
Comment (HyukjinKwon) [inline src/test/scala/com/hortonworks/spark/sql/hive/streaming/TestHiveStreaming.scala]: newline at the end ..

File: src/main/scala/com/hortonworks/spark/hive/utils/HiveIsolatedClassLoader.scala [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+  private def isHiveClass(name: String): Boolean = {
+    name.startsWith("org.apache.hadoop.hive.") ||
+      name.startsWith("org.apache.hive.") ||
+      name.startsWith("org.apache.orc.")
+  }
+
+  private def classToPath(name: String): String =
+    name.replaceAll("\\.", "/") + ".class"
+}
```
Comment (HyukjinKwon) [inline src/main/scala/com/hortonworks/spark/hive/utils/HiveIsolatedClassLoader.scala]: ditto for newline (scalastyle catches this).

File: build.sbt (hunk 1 above)
Comment (EricWohlstadter) [reply build.sbt]: indeed

File: src/main/scala/com/hortonworks/spark/hive/utils/HiveIsolatedClassLoader.scala (hunk 3 above)
Comment (EricWohlstadter) [reply src/main/scala/com/hortonworks/spark/hive/utils/HiveIsolatedClassLoader.scala]: added newline

File: src/test/scala/com/hortonworks/spark/sql/hive/streaming/TestHiveStreaming.scala (hunk 2 above)
Comment (EricWohlstadter) [reply src/test/scala/com/hortonworks/spark/sql/hive/streaming/TestHiveStreaming.scala]: added newline

**Pull request merged**